### DISCUSSION TITLE
No GLIBCXX

### DISCUSSION THREAD

**Discussion by phisanti** [explosion/spaCy#11456] [Help: Installation]
I have just installed spacy via `pip3 install spacy`. Then, when I try to run `import spacy `module I got the following error:

```
/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /home/scm/miniconda3/envs/tw_feeds/lib/python3.10/site-packages/spacy/tokens/span_group.cpython-31
0-x86_64-linux-gnu.so)
```

I am running a ubuntu 20 in a WSL machine. I have already installed the GCC libs, so not sure how to get the GLIBCXX version that it's asking for.

**Comment by adrianeboyd** [comment]
If you are using conda, can you try installing spacy through conda instead?

```shell
conda install -c conda-forge spacy
```

You can also pick `conda` in the quickstart here for the right commands: https://spacy.io/usage

I haven't come across that particular error so I'm not sure how to fix it for pip, but I am hopeful that the conda install should work.

**Comment by adrianeboyd** [comment]
Let me transfer this to the discussion board's installation section...
